benchmark/streamly-benchmarks.cabal

```diff
@@ -593,6 +594,19 @@ benchmark FileSystem.Handle
   else
     buildable: True
 
+benchmark FileSystem.DirIO
+  import: bench-options
+  type: exitcode-stdio-1.0
+  hs-source-dirs: Streamly/Benchmark/FileSystem
+  main-is: DirIO.hs
+  if flag(use-streamly-core)
+    buildable: False
```

Comment:
why is it false on streamly-core? does it use `streamly`?

Reply:
Yeah, needs streamly.